### PR TITLE
Adding files

### PR DESCRIPTION
**Pull request by Trust-007**
I added a html and css files to this branch, also added an md file which i ignored using the .gitignore file.

### CODE REVIEW

File: .gitIgnore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (leonardodiasb) [inline .gitIgnore]: - [x] Your `.gitignore` file is not working and the `test.md` file was not ignored and committed to this repository. Please, double-check for typos in the name of your files and make sure the `test.md` file is not committed to this repository.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Trust-007, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [x] Your Pull Request should have a short summary describing introduced changes in more detail. Kindly add that to this PR. You can edit the PR message by clicking in the '3 dots' on the top right corner and clicking on 'edit'.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitIgnore (hunk 1 above)
Comment (Trust-007) [reply .gitIgnore]: ok but do i close the pull request in order to make the changes

File: README.md [hunk 2]
```diff
@@ -1 +1,15 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+
+Built With
+
+    HTML
+    CSS
+
+
```
Comment (youmari) [inline README.md:8]: - [ ] [OPTIONAL] IT would be better if you add a section where you tell how the users should install your project locally

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @Trust-007 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**